securedrop/manage.py

```diff
@@ -213,27 +207,40 @@ def listdir_fullpath(d):
             os.remove(path)
 
 
-def main():
-    valid_cmds = [
-        "run",
-        "test_unit",
-        "test",
-        "reset",
-        "add_admin",
-        "clean_tmp"]
-    help_str = "./manage.py {{{0}}}".format(','.join(valid_cmds))
+def get_args():
+    parser = ArgumentParser(prog=__file__,
+                            description='A tool to help admins manage and devs hack')
+
+    subparsers = parser.add_subparsers()
+    subparsers.required = True
+    subparsers.dest = 'subcommand'
```

Comment:
The function name is already saved to the `parser.func` attribute which we call with `args.func()`. This writes the title argument of the relevant `add_parser` subcommand to `parser.subcommand`, which is unused/ unecessary. Since this line of code seems to do nothing, I suggest removing it.